lua/swenv/api.lua

```diff
@@ -148,15 +149,33 @@ local get_pyenv_base_path = function()
   end
 end
 
+local to_set = function (some_list)
+  local set = {} 
+  for _, key in ipair(some_list) do
+    set[key] = true
+  end
+  return set
+end
+
 M.get_venvs = function(venvs_path)
   local venvs = {}
   vim.list_extend(venvs, get_venvs_for(venvs_path, 'venv'))
-  vim.list_extend(venvs, get_venvs_for(get_pixi_base_path(), 'pixi'))
-  vim.list_extend(venvs, get_venvs_for(get_conda_base_path(), 'conda'))
-  vim.list_extend(venvs, get_conda_base_env())
-  vim.list_extend(venvs, get_venvs_for(get_micromamba_base_path(), 'micromamba'))
-  vim.list_extend(venvs, get_venvs_for(get_pyenv_base_path(), 'pyenv'))
-  vim.list_extend(venvs, get_venvs_for(get_pyenv_base_path(), 'pyenv', { only_dirs = false }))
+
+  local ignore_envs = to_set(settings.ignore_envs_groups)
+  if ignore_envs and not ignore_envs['conda'] then
```

Comment:
```suggestion
  if not ignore_envs['conda'] then
```
wouldn't this work as well?